test/e2e/scheduling/preemption.go

```diff
@@ -92,13 +92,29 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 		// Create one pod per node that uses a lot of the node's resources.
 		By("Create pods that use 60% of node resources.")
 		pods := make([]*corev1.Pod, len(nodeList.Items))
+		allPods, err := cs.CoreV1().Pods(metav1.NamespaceAll).List(metav1.ListOptions{})
 		for i, node := range nodeList.Items {
+			// There is a chance that node.Status.Allocatable has changed between the time we created node objects
+			// and now, so better to calculate the current usage again before assigning. This can happen when pods
+			// frequently jump from one node to another either because of eviction or preemption. So better to get
+			// current value
+			currentCpuUsage, currentMemUsage := getCurrentPodUsageOnTheNode(node.Name, allPods.Items, podRequestedResource)
 			cpuAllocatable, found := node.Status.Allocatable["cpu"]
 			Expect(found).To(Equal(true))
-			milliCPU := cpuAllocatable.MilliValue() * 40 / 100
+			milliCPU := cpuAllocatable.MilliValue()
+			if milliCPU != currentCpuUsage {
+				framework.Logf("Allocatable is not equal to current usage change it to current value for node %v", node.Name)
```

Comment:
This is almost always the case. Node allocatable is almost always larger than node resource usage.